src/wizard/javascript/gatsby.md

```diff
@@ -20,7 +20,7 @@ npm install --save @sentry/gatsby
 Register the plugin in your Gatsby configuration file (typically `gatsby-config.js`).
 
 ```javascript
-{
+module.exports = {
```

Comment:
Could you add the file name here to?

Reply:
IIRC we have other cases like that. Adding the filename changes how the code snippet renders, see even https://github.com/getsentry/sentry-docs/pull/4232/files#diff-eaab05775310590a0a9a65d2fe287113f6c72777be1b2c7af70c84b162a8a26aR27 is like that.

Reply:
Alright! Updated.